<launch>
  <arg name="INPUT" default="/timer_cam2/timer_cam_image/image_rect_color"/>
  <arg name="gui" default="true" />
  <arg name="debug_view" default="true" />
  <param name="/use_sim_time" value="true" />
  
  <node name="mask_image_to_rect"
        pkg="jsk_perception" type="mask_image_to_rect"
        output="screen"
        clear_params="true" >
    <remap from="~input" to="/timer_cam2_rec/add_mask_image/output" />
    <rosparam subst_value="true">
      rect_type: 1
    </rosparam>
  </node>

  <node name="draw_rects"
        pkg="jsk_perception" type="draw_rects">
    <remap from="~input" to="$(arg INPUT)" />
    <remap from="~input/rects" to="mask_image_to_rect/output" />
  </node>
</launch>
